### PR TITLE
Additional stack config

### PR DESCRIPTION
**Pull request by marco-hoyer**
Support configuration of service iam roles (used by cfn to manage stacks and resources) and stack policies both on a stack configuration level for all it's stacks and also per stack. Refactor other configuration properties like tags and timeout to behave equally.

### CODE REVIEW

**Comment by marco-hoyer** [comment]
@cfn-sphere/contributors any suggestions about this?

File: src/main/python/cfn_sphere/aws/cfn.py [hunk 1]
```diff
@@ -251,43 +254,64 @@ def _create_stack(self, stack):
...
-            StackName=stack.name,
-            TemplateBody=stack.template.get_template_json(),
-            Parameters=stack.get_parameters_list(),
-            TimeoutInMinutes=stack.timeout,
-            Capabilities=[
+        kwargs = {
+            "StackName": stack.name,
+            "TemplateBody": stack.template.get_template_json(),
+            "Parameters": stack.get_parameters_list(),
+            "TimeoutInMinutes": stack.timeout,
```
Comment (konz) [inline src/main/python/cfn_sphere/aws/cfn.py]: Has nothing to do with this particular change, but setting `TimeoutInMinutes` to the stack config `timout`, which uses seconds as unit confuses two different timeouts: The stack config `timeout` defines how long cfn-sphere should keep polling for events, where `TimeoutInMinutes` defines how long CloudFormation itself waits for the stack operation to finish.

`"TimeoutInMinutes": stack.timeout / 60` would be correct, but an incompatible change for many. I would just not set it, as the default timeout is not defined but depends on the type of resources used: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html

Comment (marco-hoyer) [inline src/main/python/cfn_sphere/aws/cfn.py]: thanks a lot. you're perfectly right. I mixed this up. Removed it.

Comment (konz) [inline src/main/python/cfn_sphere/aws/cfn.py]: Cool! Looks good otherwise! 👍

**Pull request merged**